Metaverse/Source/Metaverse/Public/Items/Coin.h

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022, Cronos Labs. All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameFramework/Actor.h"
+
+#include "Coin.generated.h"
+
+class UPrimitiveComponent;
+class UText3DComponent;
+class USphereComponent;
+
+UCLASS() class METAVERSE_API ACoin : public AActor {
+  GENERATED_BODY()
+
+private:
+  UPROPERTY(VisibleAnywhere, BlueprintReadOnly, Category = "Components",
+            meta = (AllowPrivateAccess = "true"))
+  UStaticMeshComponent *StaticMeshComponent;
+  UPROPERTY(VisibleAnywhere, BlueprintReadOnly, Category = "Components",
+            meta = (AllowPrivateAccess = "true"))
+  UText3DComponent *Text3D;
+  UPROPERTY(VisibleAnywhere, BlueprintReadOnly, Category = "Components",
+            meta = (AllowPrivateAccess = "true"))
+  UStaticMeshComponent *StaticMesh;
+  UPROPERTY(VisibleAnywhere, BlueprintReadOnly, Category = "Components",
```

Comment:
Components is too broad,
how about more detailed name?

Reply:
It is a physical part of the `Coin` asset, the naming is not important...

Reply:
<img width="941" alt="image" src="https://user-images.githubusercontent.com/92778185/222614789-4f915636-9890-4e67-8043-a6f3c4ab1f59.png">